pkg/grpc/server.go

```diff
@@ -156,6 +159,20 @@ func (s *server) StartNonBlocking() error {
 			internalv1pb.RegisterServiceInvocationServer(server, s.api)
 		} else if s.kind == apiServer {
 			runtimev1pb.RegisterDaprServer(server, s.api)
+
+			// HACK: Testing durabletask-go integration
+			grpcExecutorFunc := func(be backend.Backend) backend.Executor {
+				return backend.NewGrpcExecutor(server, be)
```

Comment:
The durabletask library takes as a parameter a `grpc.Server` object. This means we don't actually need to register any protos with Dapr itself. The library will take care of registering its service with the gRPC server.